### PR TITLE
Docs testing

### LINKED ISSUE #5376: update installation docs to say we test on on aarch64
https://github.com/pyca/cryptography/blob/master/docs/installation.rst

### PR DESCRIPTION
**Pull request by reaperhulk**
fixes #5376

### CODE REVIEW

**Comment by alex** [comment]
Uhh, did you intend to include #5412 in this PR?

**Comment by alex** [comment]
yeah... aarch64 needs to be added to the word list

File: docs/spelling_wordlist.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+AArch64
```
Comment (alex) [inline docs/spelling_wordlist.txt:1]:
```suggestion
AArch
```

Comment (reaperhulk) [reply docs/spelling_wordlist.txt:1]: I hate the spellchecker.

**Pull request merged**